hw/ip/tlul/data/tlul.prj.hjson

```diff
@@ -5,8 +5,12 @@
 {
     name:               "tlul",
     version:            "0.5",
-    life_stage:         "L1",
-    design_stage:       "D2",
-    verification_stage: "V2",
-    notes:              "the target for this entry is the autogenerated xbar_main within top_earlgrey",
+    life_stage:         "L2",
+    design_stage:       "D3",
+    verification_stage: "V3",
+    notes:              '''the target for this entry is the autogenerated xbar_main within top_earlgrey.
+
+                        Signoff commit id:
+                        [f723f327bae4c70a14409ab260ddcb7856be2ebd](https://github.com/lowRISC/opentitan/tree/f723f327bae4c70a14409ab260ddcb7856be2ebd)
```

Comment:
Since we have markdown now (thanks!) can we remove the uglylonghashnumber
and just link like you've done? Something like
[Signoff commit](https://github.com/lowRISC/opentitan/tree/f723f327bae4c70a14409ab260ddcb7856be2ebd). The table is already getting a little cluttery.

Also, probably best to not couple the markdown/gen_dashboard_entry items in with this signoff. Can you split that out? Consider this LGTM to be on the --> L2/D3/V3 part of the PR

Reply:
Or alternatively we could use the short form in link text:
```
$ git rev-parse --short f723f327bae4c70a14409ab260ddcb7856be2ebd
f723f32
```

I think GitHub will show this in the web UI too.